src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1098,6 +1108,21 @@ public Settings additionalSettings() {
         return builder.build();
     }
 
+    @Override
+    public List<Setting<?>> getExtensionSettings() {
```

Comment:
This looks fine to me, but I'd also like to see the usage of the setting inside the JWTVendor in `feature/extensions` branch to show how this setting impacts the payload of the JWT created of the obo token for forwarding to an extension.